gui/wxpython/gui_core/forms.py

```diff
@@ -79,6 +79,9 @@
 
 # needed when started from command line and for testing
 if __name__ == "__main__":
+    from grass.lib.gis import G_gisinit
```

Comment:
The import here is too soon. The other GRASS-related imports are only later. (This needs a fix with GRASS Python libs just being installed in the system and rest of the code can just assume they are there, but the current situation is that the code can't assume that.)

All ctypes imports need to be in `try-except ImportError` block as e.g. in `gui/wxpython/nviz/wxnviz.py`. The idea is that the GUI components start and work even if ctypes are broken for some reason. Only the parts which need to work can fail, ideally gracefully. (Standalone NVIZ does not have to work, but main GUI should.)

Reply:
I made a better version – import is done only before use. As G_gisinit("") is safe to call after initialization has been already done, calling it one more time wouldn't harm.
When  a separate C module for signature file handling will be made, this code will have to change. In the meantime – it is better than failing GUI.